synapse_data_warehouse/synapse/tables/V2.19.1__add_descriptions_to_teammember_latest.sql

```diff
@@ -0,0 +1,12 @@
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP
+
+ALTER TABLE TEAMMEMBER_LATEST SET COMMENT = 'The latest snapshot with information for team members in Synapse.';  
+
+COMMENT ON COLUMN NODE_LATEST.change_timestamp IS 'The time when any change to the team was made (e.g. update of the team or a change to its members).';
```

Comment:
This needs to be updated

Reply:
Thanks - I glossed over this

Reply:
One nuance about schemachange is that the versions have to run in order

Reply:
I used 2.19.1 because I saw Bryan's was 2.19.0. Will that work if his is merged first?

Reply:
@BWMac yes, i meant that we have to merge Bryan's PR first or else if we merged your first, schemachange will just ignore the "2.19.0" version